### PR TITLE
docs(102): initial commit

### PR DESCRIPTION
**Pull request by alexcg1**

### CODE REVIEW

**Comment by alexcg1** [comment]
@pswu11 do you think we should add a section with next steps?

File: chapters/102/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+### Inside a Flow
+
+Here, we’ll use the indexing Flow as an example:
+
+![](images/flow_pods.svg)
+
+In our Flow, different Pods perform different tasks. Documents (in this case a book) are:
+
+1. Segmented into chunks (sentences)
+2. Transformed into vector embeddings
```
Comment (JoanFM) [inline chapters/102/index.md]: the chunks are transformed into vector embeddings

File: chapters/102/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+3. Indexed in key-value pairs
```
Comment (JoanFM) [inline chapters/102/index.md]: Indexed and saved are the same thing here

**Comment by alexcg1** [comment]
Thanks for the feedback @JoanFM! Fixes done.

Quick note for any more feedback: We had **extensive** feedback sessions with @nan-wang for 101 and 102. So (because we're all so busy), it would be great if only @nan-wang could give feedback on these docs (from a technical pov).

Last time we had everyone giving feedback there were multiple engineers with *slightly different interpretations* and we got stuck in edit tennis for days. Because release is coming in just a few days let's get this out the door. Then if it needs more tweaks we can leave those to post-1.0

File: chapters/102/index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+![](images/cross_modal.svg)
+
+
+### Multi-modal search
+
+You can use multiple modalities as input for your search applications. For example, a search query input can be an image plus some descriptive text.
+
+![](images/multi_modal.svg)
+
+To implement cross-modality or multi-modality, visit the [dedicated section](link) in our documentation.
```
Comment (Yongxuanzhang) [inline chapters/102/index.md]: The link is missing is it because we haven't this section? Same for others.

File: chapters/102/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+
+### Core
+
+You construct and manage your search workflows in Core. It consists of distributed microservices (Flows, Peas, Pods, and Executors, etc.) Optionally, you can use JinaD to orchestrate these microservices.
+
+
+### Hub
+
+[Jina Hub]() for hosting Jina Pods and Apps via container images. You can upload and share images of Pods with other developers in the community.
```
Comment (Yongxuanzhang) [inline chapters/102/index.md]: The link and the following 2 are missing as well.

**Review by Yongxuanzhang — COMMENTED** [review]
Great work! Learned a lot when reading it! 😄 
Some links are missing.

File: chapters/102/index.md (hunk 4 above)
Comment (alexcg1) [reply chapters/102/index.md]: Yup. Until docs URLs are finalized there's not much we can do in that regard (unless the platform offers some other kind of linking?)

File: chapters/102/index.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
+# Jina 102
```
Comment (FionnD) [inline chapters/102/index.md]: For the purpose of a readable table of contents, should we add something like:

_Jina 102 - Understanding how basic components work together?_

What do you think?

File: chapters/102/index.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+
+### Hub
+
+[Jina Hub]() for hosting Jina Pods and Apps via container images. You can upload and share images of Pods with other developers in the community.
+
+
+### Dashboard
+
+[Jina Dashboard]() is a low-code monitoring and management environment for Jina. With Dashboard you can:
```
Comment (FionnD) [inline chapters/102/index.md]: Purely a suggestion, but you could conisder updating these descriptions with the ones from the website. So there is some brand consistency  https://docs.google.com/document/d/1DVSUMRye5icmtwGCRBpwmvKgKtZOiczdVSH0NRg6xvQ/edit#

**Review by FionnD — APPROVED** [review]

File: chapters/102/index.md (hunk 6 above)
Comment (alexcg1) [reply chapters/102/index.md]: I think @pswu11 already discussed this with Lucia

File: chapters/102/index.md (hunk 5 above)
Comment (alexcg1) [reply chapters/102/index.md]: I'm down with that. Maybe even simpler: *Jina 102: How basic components work together*

Comment (alexcg1) [reply chapters/102/index.md]: Your the docs guy, you can choose which is best :)

File: chapters/102/index.md (hunk 6 above)
Comment (pswu11) [reply chapters/102/index.md]: @FionnD I'll adjust them again based on new descriptions on the website. The reasons why I didn't copy completely from the website: 

- website is for marketing purpose (with more decorations and usually long sentences)
- in docs.jina.ai especially 102, it should be more fact-based so I tried to simplify them

File: chapters/102/index.md (hunk 5 above)
Comment (pswu11) [reply chapters/102/index.md]: @alexcg1 in this case we can adjust both 101 & 102 so it's clear by looking at the titles?

Jina 101: Basic components 
Jina 102: How basic components work together

File: chapters/102/index.md (hunk 6 above)
Comment (FionnD) [reply chapters/102/index.md]: Ah ok! Then I don't want to step on anyone toes. It was just a suggestion :)

File: chapters/102/index.md (hunk 5 above)
Comment (FionnD) [reply chapters/102/index.md]: I love that @pswu11! Let's go with that for both of them!

File: chapters/102/index.md (hunk 6 above)
Comment (pswu11) [reply chapters/102/index.md]: @FionnD :-) no worries my toes are strong (??)

**Comment by FionnD** [comment]
@alexcg1 @pswu11 same question, what's needed to merge this? (and will it be done by tomorrow :D )

**Comment by pswu11** [comment]
@FionnD it's done now, just changed from md to rst. We'll need to get TOC updated to see this in our TOC tho? Will you do that after it's merged?

**Review by pswu11 — APPROVED** [review]

File: chapters/102/index.rst [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+1. Input Documents segmented into chunks (sentences)
+2. Chunks transformed into vector embeddings
+3. Chunks indexed in key-value pairs and saved to storage
+
+Inside a Pod
+----------------------
+
+.. image:: /chapters/102/images/pod_peas.png 
+
+In the Segmenter Pod, there’s only one Pea. But sometimes you group multiple copies of the same Peas in a Pod to increase reliability or performance. To implement sharing and parallelization, check `this section`_.
```
Comment (bwanglzu) [inline chapters/102/index.rst]: one comment is that we should link terms such as `Pod`, `Flow` to our glossary, checkout [here](https://docs.jina.ai/master/chapters/glossary/glossary.html) to see Jina glossaries, and [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html) to find how how to do it.

**Comment by FionnD** [comment]
I will update the TOC. Please merge after Bo comment is fixed.

**Comment by alexcg1** [comment]
- ToC - I see @FionnD is on this :)
- Image size - could we shrink them? And add some padding if possible
- Image cropping - big white padding around our cartoon characters. Can we crop?

Nothing urgent and no showstoppers :)

File: chapters/102/index.rst (hunk 7 above)
Comment (pswu11) [reply chapters/102/index.rst]: @bwanglzu Since Flow Pod & Pea appear too many times in 102, I would only link them for the first time they appear.

Comment (bwanglzu) [reply chapters/102/index.rst]: sounds good to me! it's a minor thing so I didn't "request change". Feel free to merge.

**Comment by pswu11** [comment]
@alexcg1 fixed the space and resized the graphics :-)

File: chapters/102/index.rst (hunk 7 above)
Comment (pswu11) [reply chapters/102/index.rst]: @bwanglzu I added terms in a few places. Feel free to double check.

**Comment by alexcg1** [comment]
LGTM :+1:

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: chapters/102/index.rst [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+|
+
+.. image:: /chapters/102/images/pea_exec_driver.png 
+	:width: 60%  
+
+|
+
+As you can see, a Pea is a wrapper for an :term:`Executor` and its Driver. 
+
+While Peas and Pods perform the communication in a Flow, the messages themselves come from the Executors. And each Executor needs a specific Driver to handle its input and output messages.
```
Comment (JoanFM) [inline chapters/102/index.rst:98]: the messages are processed by Executors may be more accurate than saying they come from?

**Pull request merged**